odoo/addons/base/ir/ir_model.py

```diff
@@ -912,9 +912,12 @@ def _add_manual_fields(self, model):
         fields_data = self._get_manual_field_data(model._name)
         for name, field_data in fields_data.items():
             if name not in model._fields and field_data['state'] == 'manual':
-                field = self._instanciate(field_data)
-                if field:
-                    model._add_field(name, field)
+                try:
+                    field = self._instanciate(field_data)
+                    if field:
+                        model._add_field(name, field)
+                except Exception:
+                    _logger.exception("Failed to load field %r of model %r: skipped", field_data['name'], model._name)
```

Comment:
This is a technical message.  I suggest to use *modelname*.*fieldname* to denote fields.
That's easy enough to read and very easy to *grep* :wink: 
```suggestion
                    _logger.exception("Failed to load field %s.%s: skipped", model._name, field_data['name'])
```

Reply:
@rco-odoo thank you, updated; runbot is green